### PR TITLE
update-layout-colors: add_files and review_work

### PR DESCRIPTION
**Pull request by srjfoo**
add_files.php and review_work.php still had inline styles for status colors. Moved those to classes and defined the background colors in layout.less.

Test in [update-layout-colors](https://www.pgdp.org/~srjfoo/c.branch/update-layout-colors/) sandbox.

### CODE REVIEW

File: tools/project_manager/add_files.php [hunk 1]
```diff
@@ -824,12 +824,12 @@ function display()
                     echo "</td>";
 
                     // action
-                    $bgcolors = array(
-                        ''        => '#ffffff',
-                        'add'     => '#ccffcc',
-                        'replace' => '#ffccaa',
-                        'same'    => '#ffffff',
-                        'error'   => '#ffcccc',
+                    $bgclass = array(
```
Comment (cpeel) [inline tools/project_manager/add_files.php:827]: Right now this class just changes the background of the element, but it isn't restricted to only that. Consider renaming this just `$class` or `$addl_class` or somesuch to separate it from the "background" aspect.

File: tools/proofers/review_work.php [hunk 2]
```diff
@@ -446,9 +446,13 @@ function _echo_round_select($rounds,$selected) {
     // are highlighted as that data doesn't really tell me much
     // ... but it shows the evaluator which projects to look at
     // ... but not necessary with revamp of page
+    //
```
[the remark this answers is not in the document]
Comment (srjfoo) [reply tools/proofers/review_work.php:449]: I was wondering about that, thanks.

File: tools/project_manager/add_files.php (hunk 1 above)
Comment (srjfoo) [reply tools/project_manager/add_files.php]: Done.

File: tools/proofers/review_work.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (srjfoo) [reply tools/proofers/review_work.php]: Done.

File: tools/proofers/review_work.php [hunk 3]
```diff
@@ -446,9 +446,13 @@ function _echo_round_select($rounds,$selected) {
...
+    // If it is necessary to have a background assigned to $n_latered_bg
+    // at some point, use $n_w_diff_bg as an example: this should be set
+    // up as a class, not an inline style.
     
     $n_latered_bg = ( $n_latered   > 0 ? "" : "" );
-    $n_w_diff_bg  = ( $n_with_diff > 0 ? "style='background-color: #ccffcc;'" : "" );
+    $n_w_diff_bg  = ( $n_with_diff > 0 ? "has-diff" : "" );
```
[the remark this answers is not in the document]
Comment (srjfoo) [reply tools/proofers/review_work.php]: Done.

**Pull request merged**